AutoGear.lua

```diff
@@ -2160,7 +2173,7 @@ AutoGearFrame:SetScript("OnEvent", function (this, event, arg1, arg2, arg3, arg4
 		if (AutoGearDB.AutoRepair == true) then
 			-- repair all gear
 			local cashString = AutoGearCashToString(GetRepairAllCost())
-			if not (IsClassic or IsTBC or IsWotLK) then
+			if TOC_VERSION_CURRENT > 20300 then
```

Comment:
What's the rationale for 20300 over 20000?  Was this only introduced in 20300?

Reply:
yes